docs/native/src/faq/filetypes.rst

```diff
@@ -0,0 +1,16 @@
+What file types are supported in |Office iOS|?
+============================================== 
+
+PPT
+   @"pptx", @"ppsx", @"ppt", @"pps", @"pptm", @"potm", @"potx", @"ppsm", @"pot", @"thmx", @"odp"
```

Comment:
yeah we can probably remove the @. there's no reason for them to be there. It was a direct cut and paste from code devs provided.